src/main/java/org/jenkinsci/main/modules/sshd/SSHD.java

```diff
@@ -85,10 +82,8 @@ public synchronized void start() throws IOException, InterruptedException {
         sshd.setUserAuthFactories(Arrays.<NamedFactory<UserAuth>>asList(new UserAuthNamedFactory()));
                 
         sshd.setCipherFactories(Arrays.asList(// AES 256 and 192 requires unlimited crypto, so don't use that
```

Comment:
Err, we could, but I don't think anyone will really benefit of using AES-192 in SSH connection. CBC modes drop is far more important and that's the purpose of this pull request.